### ISSUE TITLE
Allow custom template.nuspec

### ISSUE THREAD

**Issue by fasterthanlime** [atom/grunt-electron-installer#89]
Clearly marked as advanced feature, yada yada, even undocumented for now if maintainers wish, but there's two issues preventing windows installer builds on Linux in the current template:

  * Usage of backspace in paths, means globs like `resources\**\*` yield 0 results, all the app's files are missing - replacing with `/` fixes that.
  * Inclusion of extension-less files like `LICENSE`, generates an invalid `[Content-Types].xml` file, cf. https://github.com/Squirrel/Squirrel.Windows/issues/520

ideally grunt-electron-installer would work around these out of the box, but in the meantime being able to pass a path to a `template.nuspec` file to be used would be very very cool.

**Comment by fasterthanlime** [comment]
Implemented here: https://github.com/atom/grunt-electron-installer/compare/master...itchio:patch-custom-nuspec — waiting for Squirrel.Windows changes to land before opening PR

**Comment by paulcbetts** [comment]
The custom nuspec might be interesting, but we should also make grunt-electron-installer silently fix this up

**Comment by fasterthanlime** [comment]
> we should also make grunt-electron-installer silently fix this up

Right, but I saw no good immediate option (either use regular expressions and risk overshooting, or parse xml + re-emit it and then you have a dependency on an XML parser, an XML generator :/)

Or.. two sets of template? Or make the whole path list be a 'variable' in the template and generate that from the task itself?

The part that annoys me the most is having to drop 'LICENSE' because of that NuGet bug that only happens on Mono. :cry:

**Comment by paulcbetts** [comment]
> The part that annoys me the most is having to drop 'LICENSE' because of that NuGet bug that only happens on Mono. :cry:

Yeah :-/
